.github/workflows/lint.yml

```diff
@@ -7,15 +7,47 @@ on:
       - develop
 
 jobs:
-  formatting:
+  formatting-juia:
+    if: "!contains(github.event.head_commit.message, 'skip_ci')"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.3.0]
```

Comment:
We use julia 1.5 in the [test.yml](https://github.com/Breakthrough-Energy/REISE.jl/blob/develop/.github/workflows/test.yml) workflow, should we stick with that for consistency?

Reply:
Done.